fcrepo-import-export-driver/src/main/java/org/fcrepo/importexport/ArgParser.java

```diff
@@ -37,78 +49,145 @@
  */
 public class ArgParser {
 
-    final private Options cmdOptions;
+    private static final Logger logger = getLogger(ArgParser.class);
 
     public static final String DEFAULT_RDF_EXT = ".ttl";
     public static final String DEFAULT_RDF_LANG = "text/turtle";
+    public static final String CONFIG_FILE_NAME = "config.txt";
```

Comment:
Should we have a more descriptive config filename like `importexport.config`?  Do we expect this file to live in a directory along with the import-export JAR file, where it will be obvious what it is config for?